services/spar/test-integration/Test/LoggingSpec.hs

```diff
@@ -0,0 +1,19 @@
+module Test.LoggingSpec (spec) where
+
+import Imports
+import Control.Lens
+import Data.String.Conversions (ST)
+import Spar.App
+import System.Logger as Log
+import System.IO.Silently (capture)
+import Util
+
+
+spec :: HasCallStack => SpecWith TestEnv
+spec = describe "logging" $ do
+  it "does not log newlines" $ do
```

Comment:
Would be nice to leave a comment explaining why newlines are not logged (it's not obvious that `simpleDefSettings` does that by default).

Reply:
https://gitlab.com/fisx/tinylog/blob/settings-auxiliaries/src/System/Logger/Settings.hs#L212-219

is that enough?  or does it have to be in this test?

Reply:
I'd rather have it in the test. My reasoning is that if `simpleDefSettings` changes and this test starts to fail, a comment that immediately points to `simpleDefSettings` would be helpful. Otherwise it's kinda hard to figure out where exactly the newlines were _supposed_ to be stripped (in the tinylog library? in our settings? in some monad we have around? or perhaps we intercept stdout and do something to it?)

Reply:
done.  (if you still want something else: would you like to make a commit yourself?)